Source/Core/DiscIO/CMakeLists.txt

```diff
@@ -66,7 +66,7 @@ add_library(discio
 
 target_link_libraries(discio
 PUBLIC
-  common
+  core
```

Comment:
Why not keep the `common` line?

Reply:
`core` depends on `common` already, so it's redundant.  But there doesn't seem to be any problem with keeping it either.

Reply:
Yeah, I just figured there could be a point in avoiding implicit dependencies. But I don't know what our typical stance is on this in CMake.

Reply:
DolphinQt only specifies `core` and not `common` so I think implicit/transitive dependencies are fine.

Reply:
Yeah, seems consistent with the other targets and `core` will always depend on `common`. That said, I think our policy should be to avoid transitive dependencies in general (e.g. if both `core` and `discio` use `fmt` then they should both depend on the fmt target explicitly -- `discio` should not be relying on `core` to depend on `fmt::fmt`). `core`/`common` would be an exception to the rule.